src/components/organisms/EventModal/EventModal.tsx

```diff
@@ -68,15 +70,17 @@ export const EventModal: React.FC<EventModalProps> = ({
   return (
     <Modal show={show} onHide={onHide} className="EventModal">
       <div className="EventModal__content">
-        <h4 className="EventModal__title">{event.name}</h4>
+        <h4 className="EventModal__title">
```

Comment:
As above, why do we want to render markdown in titles?

Reply:
no more